views/Products/Create/Form.tsx

```diff
@@ -32,25 +57,51 @@ const CreateProductForm: FC<MutableProductFormProps> = ({ onSubmit, loading }) =
         <input
           className="input-product font-visby"
           type="text"
+          id="basePriceSol"
           placeholder="00.00"
-          {...register('basePriceSol', { required: true })}
+          {...register('basePriceSol', { required: { value: true, message: 'Debe colocar un precio' } })}
         />
-        {errors.basePriceSol && <small className="text-red-500">{errors.basePriceSol.message}</small>}
       </div>
+      {errors.basePriceSol && <small className="text-red-500">{errors.basePriceSol.message}</small>}
+
       <label className="text-size label" htmlFor="description">
         Descripción del producto
       </label>
       <textarea
         className="textarea font-visby"
         id="description"
         placeholder="Escribe aquí..."
-        {...register('description', { required: true })}
+        {...register('description', { required: { value: true, message: 'El campo no puede estar vacio' } })}
       />
       {errors.description && <small className="text-red-500">{errors.description.message}</small>}
 
       <label className="text-size label">Añadir imagen del producto</label>
-      <label className="image font-visby">Añadir imagen</label>
+      <div>
+        <div className={stylesInput['container-input']}>
+          <input
+            id="image-file"
+            type="file"
+            accept="image/*"
+            className={stylesInput['input-file']}
+            {...register('image')}
+            onChange={handleFile}
+          />
+          <label htmlFor="image-file" className={stylesInput.image}>
+            <figure>
+              <Image
+                width={imageUrl ? 300 : 40}
+                height={imageUrl ? 200 : 40}
+                objectFit="contain"
+                src={imageUrl || 'https://img.icons8.com/ios/100/image.png'}
+                alt="Imagen del evento"
+              />
+            </figure>
+            <span className={stylesInput.label}>{imageFile?.name ? 'Cambiar imagen' : 'Añadir imagen'}</span>
+          </label>
+        </div>
+      </div>
```

Comment:
When I try to edit a product it works properly, but the image can't be changed and is empty when should be the image that I added to the product before
![lastrecord](https://user-images.githubusercontent.com/66505715/190203212-193a289c-1819-4100-acc2-333e07f3abd6.gif)